### PR TITLE
Move rename files

### PR DESCRIPTION
**Pull request by adhusson**
| before | after |
|--|--|
| `Constant.t.sol` | `Constants.t.sol` |
| `BinLib.sol` | `TickTreeLib.sol` |
| `MgvStructs.post.sol` | `Structs.post.sol` |
| `MgvType.pre.sol.ts` | `Struct.pre.sol` |
| `MgvTypeTest.pre.sol.ts` | `StructTest.pre.sol` |
| `Mgv*.post.sol` | `*.Post.sol` |
| `src/*` to be audited | `src/core/*` |
| `lib/*` to be audited | `lib/core/*` |
| `mgv_src/IERC20.sol` | `mgv_lib/IERC20.sol` |

### CODE REVIEW

File: src/preprocessed/README.md
Comment (espendk) [inline src/preprocessed/README.md:1]: Was this intentionally deleted instead of moved?

File: test/preprocessed/GlobalTest.post.sol [hunk 1]
```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier:	AGPL-3.0
+
+pragma solidity ^0.8.13;
+
+import "mgv_lib/Test2.sol";
+import "mgv_src/core/MgvLib.sol";
+
+// Warning: fuzzer will run tests with malformed packed arguments, e.g. bool fields that are > 1.
+
+contract MgvGlobalTest is Test2 {
```
Comment (espendk) [inline test/preprocessed/GlobalTest.post.sol]: Filename and contract name are inconsistent

File: test/preprocessed/GlobalTest.post.sol
Comment (espendk) [inline test/preprocessed/GlobalTest.post.sol:1]: I assume this was simply extracted from another file, so I haven't reviewed the contents. Let me know if that's not the case.

File: test/preprocessed/LocalTest.post.sol [hunk 2]
```diff
@@ -0,0 +1,316 @@
+// SPDX-License-Identifier:	AGPL-3.0
+
+pragma solidity ^0.8.13;
+
+import "mgv_lib/Test2.sol";
+import "mgv_src/core/MgvLib.sol";
+
+// Warning: fuzzer will run tests with malformed packed arguments, e.g. bool fields that are > 1.
+
+contract MgvLocalTest is Test2 {
```
Comment (espendk) [inline test/preprocessed/LocalTest.post.sol]: Inconsistent naming here as well.

File: test/preprocessed/LocalTest.post.sol
Comment (espendk) [inline test/preprocessed/LocalTest.post.sol:1]: I assume this was simply extracted from another file, so I haven't reviewed the contents.

File: test/preprocessed/OfferDetailTest.post.sol [hunk 3]
```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier:	AGPL-3.0
+
+pragma solidity ^0.8.13;
+
+import "mgv_lib/Test2.sol";
+import "mgv_src/core/MgvLib.sol";
+
+// Warning: fuzzer will run tests with malformed packed arguments, e.g. bool fields that are > 1.
+
+contract MgvOfferDetailTest is Test2 {
```
Comment (espendk) [inline test/preprocessed/OfferDetailTest.post.sol]: and here

File: test/preprocessed/OfferDetailTest.post.sol
Comment (espendk) [inline test/preprocessed/OfferDetailTest.post.sol:1]: I assume this was simply extracted from another file, so I haven't reviewed the contents.

File: test/preprocessed/OfferTest.post.sol
Comment (espendk) [inline test/preprocessed/OfferTest.post.sol:1]: I assume this was simply extracted from another file, so I haven't reviewed the contents.

File: test/preprocessed/OfferTest.post.sol [hunk 4]
```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier:	AGPL-3.0
+
+pragma solidity ^0.8.13;
+
+import "mgv_lib/Test2.sol";
+import "mgv_src/core/MgvLib.sol";
+
+// Warning: fuzzer will run tests with malformed packed arguments, e.g. bool fields that are > 1.
+
+contract MgvOfferTest is Test2 {
```
Comment (espendk) [inline test/preprocessed/OfferTest.post.sol]: and here

**Review by espendk — COMMENTED** [review]
Nice! 

Added some comments

File: src/preprocessed/README.md
Comment (adhusson) [reply src/preprocessed/README.md:1]: no not at all

File: test/preprocessed/GlobalTest.post.sol
Comment (adhusson) [reply test/preprocessed/GlobalTest.post.sol:1]: yes this is all the result of a preprocessor

**Review by espendk — APPROVED** [review]
lgtm

**Pull request merged**